sstables/mc/parquet_writer.hh

```diff
@@ -0,0 +1,286 @@
+
+#pragma once
+
+#include <seastar/parquet/parquet/column_writer.h>
+#include <mutation_fragment.hh>
+#include <tombstone.hh>
+#include <dht/i_partitioner.hh>
+#include <range_tombstone.hh>
+#include <seastar/parquet/parquet/file_writer.h>
+#include <seastar/parquet/parquet/schema.h>
+#include <seastar/parquet/parquet/properties.h>
+#include <seastar/core/sstring.hh>
+#include <schema.hh>
+#include <iostream>
+
+namespace parquet {
+
+// TODO will be removed
+std::ofstream _trace = std::ofstream("/tmp/parquet-trace.log", std::ofstream::binary);
```

Comment:
This is a "homemade" way for debug logging (since gdb is horribly slow). I will be glad if you share a more preferred way